Doc/using/cmdline.rst

```diff
@@ -260,12 +260,23 @@ Miscellaneous options
 
 .. cmdoption:: -O
 
-   Turn on basic optimizations.  See also :envvar:`PYTHONOPTIMIZE`.
+   Remove assert statements and any code conditional on the value of
+   :const:`__debug__`.  Change the filename extension for compiled
+   (:term:`bytecode`) files to include ``.opt-1`` before the ``.pyc``
```

Comment:
I will change these to read 'Augment the filename for compiled bytecode files by adding .opt-1 before the .pyc extension. '  It does not matter whether one thinks of the addition as a 2nd extension or not, and the change is strictly an addition.